patient-care/webapp/client/manageObjects/manageSampleGroups.html

```diff
@@ -65,55 +69,65 @@
     </a>
   </h3>
 
-  {{>sampleGroupExprVarFilters}}
+  {{> sampleGroupExprVarFilters}}
+
+  {{> samplesAndFeatures}}
 
   <h3>Filters applied to data sets</h3>
 
   <div class="ui list">
-    {{#each data_sets}}
-      <div class="item">
-        <div class="content">
-          <div class="header">{{data_set_name}}</div>
-          {{#if length filters}}
-            Filtered to {{sample_count}}
-            {{> optionalS word="sample" count=sample_count}}
-            from
-            {{unfiltered_sample_count}}.
-          {{else}}
-            All {{unfiltered_sample_count}} samples included without filtering.
-          {{/if}}
-          {{> listSamplesButton sample_labels}}
+    {{#let sampleGroupName=name}}
```

Comment:
This whole block changed because I added this `#let` up here.  It's used in finding the `filename` for the samples download button.